docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po

```diff
@@ -0,0 +1,207 @@
+#, fuzzy
+msgid "Create a recovery device modal"
+msgstr "Dialogue de création d'un appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:38
+msgid ""
+"Click ``Next``, you will be prompted to enter the password for the device "
+"you have selected"
+msgstr ""
+"Cliquer sur ``Suivant``, vous serez demandé d'entrer le mot de passe pour "
+"l'appareil que vous avez sélectionné"
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Create a recovery device re-authenticate modal"
+msgstr ""
+"Dialogue de reconnection pour la création d'un appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:43
+msgid ""
+"After confirmation, the recovery file is created and a passphrase is "
+"displayed. **You need to save the passphrase** as it will not be displayed "
+"again."
+msgstr ""
+"Après confirmation, le fichier de récupération est créé et la phrase de "
+"passe est affichée. **Vous avez besoin de sauvegarder la phrase de passe** "
+"car c'est la seul fois qu'elle sera affichée."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Create a recovery device confirmation modal"
+msgstr "Dialogue de confirmation de création d'appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:50
+msgid "The passphrase should be handled carefully like any other password."
+msgstr ""
+"La phrase de passe doit être manipulée comme n'importe quels mot de passe."
+
+#: ../../userguide/recovery_device.rst:52
+msgid ""
+"You can print out the passphrase and put-it in a safe to secure it. "
+"Nevertheless, you should not save it alongside the recovery file."
+msgstr ""
+"Vous pouvez imprimer la phrase de passe et la mettre dans un coffre-fort "
+"pour la sécuriser. Néanmoins, vous ne devez pas la sauvegarder à coté du "
+"fichier de récupération."
+
+#: ../../userguide/recovery_device.rst:57
+msgid "Use a recovery device"
+msgstr "Utiliser un appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:59
+msgid "In the Device recovery dialog, select ``Recover a Device``"
+msgstr ""
+"Dans le dialogue de récupération d'appareil, sélectionner ``Récupérer un "
+"appareil``"
+
+#: ../../userguide/recovery_device.rst:63
+msgid "Select the recovery file of the device you want to recover."
+msgstr ""
+"Sélectionner le fichier de récupération de l'appareil que vous voulez "
+"récupérer."
+
+#: ../../userguide/recovery_device.rst:64
+msgid "Enter the passphrase for the recovery device."
+msgstr "Entrer la phrase de passe de l'appareil de récupération."
+
+#: ../../userguide/recovery_device.rst:65
+msgid "Enter the name of the new device."
+msgstr "Entrer le nom du nouvel appareil."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Recover a device modal"
+msgstr "Dialogue de récupération d'appareil"
+
+#: ../../userguide/recovery_device.rst:70
+msgid ""
+"Click ``Next``, you will be prompted to enter a new password and confirm it."
+msgstr ""
+"Cliquer sur ``Suivant``, vous serez demandé d'entrer un nouveau mot de "
```

Comment:
```suggestion
"Cliquer sur ``Suivant``, et entrez un nouveau mot de "
```

Reply:
Why `entrez` is conjugated ? `Cliquer` is not.

```suggestion
"Cliquer sur ``Suivant``, et entrer un nouveau mot de "
```

Reply:
Yep, that was a typo. your suggestion is what I meant

Reply:
Ou meme: 
```suggestion
"Cliquer sur ``Suivant``, puis entrer un nouveau mot de "
```

Reply:
done